lib/rouge/lexers/php.rb

```diff
@@ -65,14 +68,17 @@ def builtins
       def self.keywords
         @keywords ||= Set.new %w(
           and E_PARSE old_function E_ERROR or as E_WARNING parent eval
-          PHP_OS break exit case extends PHP_VERSION cfunction FALSE
+          PHP_OS break exit case extends PHP_VERSION cfunction false
```

Comment:
I agree that it doesn't make sense to have `false` (and elsewhere `true` and `null`) in the keywords group. Please feel free to pull these out as you suggested and instead tokenise them as constants.